lib/recurly/element/cvv.js

```diff
@@ -0,0 +1,11 @@
+import Element from './element';
+
+export function factory (options) {
+  return new CvvElement({ ...options, elements: this });
+}
+
+export class CvvElement extends Element {
```

Comment:
Debating whether or not this calls for a new element class or we can add `supportsTokenization` to `CardCvvElement` to allow it to work stand alone.